app/controllers/katello/api/v2/products_controller.rb

```diff
@@ -26,6 +26,7 @@ class Api::V2::ProductsController < Api::V2::ApiController
     param :name, String, :desc => N_("Filter products by name")
     param :enabled, :bool, :desc => N_("Filter products by enabled or disabled")
     param :custom, :bool, :desc => N_("Filter products by custom")
+    param :include_available_products, :bool, :desc => N_("Whether to include available products attribute in results")
```

Comment:
If the attribute is available_content why not call the parameter that? Seems confusing to call the products API with an 'available_products' parameter and get back products but with some attribute about available products...